src/d3a/models/strategy/load_hours.py

```diff
@@ -169,8 +169,23 @@ def update_state(self):
         self._post_first_bid()
         if self.area.current_market:
             self._cycled_market.add(self.area.current_market.time_slot)
+
+        # Provide energy values for the past market slot, to be used in the settlement market
+        self.set_real_energy_of_last_market()
         self._delete_past_state()
 
+    def set_real_energy_of_last_market(self):
+        """Set the (simulated) actual energy of the device in the previous market slot."""
+        if self.area.current_market:
+            self._set_real_energy_kWh(self.area.current_market.time_slot)
+
+    def _set_real_energy_kWh(self, time_slot: DateTime) -> None:
+        """Set the (simulated) actual energy consumed by the device in a market slot."""
+        energy_forecast_kWh = self.state.get_desired_energy_Wh(time_slot) * 1000
```

Comment:
Just while I am seeing this: 
```suggestion
        energy_forecast_kWh = self.state.get_desired_energy_Wh(time_slot) / 1000.
```

Reply:
Oh wow, thanks for noticing this!

Reply:
Why do you recommend using the `.`?

Reply:
Hmm, good question. This i burned into my brain. I remember having issues with this and data types in my studies. But this was python2. I just checked and even if `self.state.get_desired_energy_Wh(time_slot)` returns an int, the devision always returns a floating point number. So, all good and I learned something. Thanks a lot!

Reply:
Interesting how experience in pain while debugging results into habits (I always add a point to the divisor). But from now on I can save "the effort".

Reply:
Haha yes, I remember it also from the past. When you suggested it I thought that it was necessary again and my nightmares were coming back! :P